### PR TITLE
fix(package): up version and change scope/name

### PR DESCRIPTION
**Pull request by brandonros**

### CODE REVIEW

**Review by eheikes — APPROVED** [review]
I'll let @brandonros merge this if we want to keep the `@banno` prefix, or modify the PR further if we want to make it a public package. I think others could find it useful, so I'd lean towards public.

**Comment by brandonros** [comment]
I like having it internal for a clarity perspective. It's more clear in my mind to know that anything Vault + npm package wise is under the `@banno` scope, instead of scattered throughout. :)

**Pull request merged**
